framework/asserts/common.py

```diff
@@ -0,0 +1,48 @@
+from hamcrest import assert_that, is_, contains_string
+
+
+def assert_status_code(response, expected_status_code):
+    """
+    Asserts that the actual status code matches the expected status code.
+
+    :param response: The response object from the API call.
+    :param expected_status_code: The expected status code.
```

Comment:
Let's go docstring by Google:

`
"""Describe

Args:
    response: The response object from the API call.
    expected_status_code: The expected status code.
"""
`

Reply:
Fixed.